packages/terra-table/src/TableCell.jsx

```diff
@@ -7,17 +7,29 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   content: PropTypes.any.isRequired,
+  /**
+   * @private
+   * Id of the helper text identifying a row is selectable. Populates the aria-describedby attribute
+   * of the child cells in this row.
+   */
+  selectRowHelpTextId: PropTypes.string,
+};
+
+const defaultProps = {
+  selectRowHelpTextId: undefined,
 };
 
 const TableCell = ({
   content,
+  selectRowHelpTextId,
   ...customProps
 }) => (
-  <td {...customProps} data-terra-table-cell>
+  <td role="gridcell" {...customProps} aria-describedby={customProps.onKeyDown ? selectRowHelpTextId : undefined} data-terra-table-cell>
```

Comment:
Why is this checking for onKeyDown callback? I would think this should be determined by if the row has `isSelectable` set to true. Perhaps in TableRow only clone child with selectRowHelpTextId if `isSelectable`

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/16a40cbff81110b7b731b0f88439ee8a2d79779c